<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="loanType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Business</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Business</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mortgage</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vehicle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wedding</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Refinancing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Major Purchases</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vacation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medical Treatment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Education</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Personal Expenses</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="creditScoreSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>850</number>
    </property>
    <property name="value">
     <number>650</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="paymentFrequency">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Monthly</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Payment Frequency</string>
    </property>
    <item>
     <property name="text">
      <string>Monthly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Biweekly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Weekly</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnGetLoan">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>570</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get a loan offer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Loan Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Loan Term:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Loan Amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Payment Frequency:</string>
    </property>
   </widget>
   <widget class="QSlider" name="loanTermSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>12</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="loanAmountSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1500000</number>
    </property>
    <property name="singleStep">
     <number>1000</number>
    </property>
    <property name="pageStep">
     <number>10000</number>
    </property>
    <property name="value">
     <number>60000</number>
    </property>
    <property name="sliderPosition">
     <number>60000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="creditScoreValue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="loanTermValue">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="loanAmountValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="minCreditScoreValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="maxCreditScoreValue">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="maxLoanTermValue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="minLoanTermValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="minLoanAmountValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="maxLoanAmountValue">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>months</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MDL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
